src/main/views/subscription-search-case-results.njk

```diff
@@ -0,0 +1,50 @@
+{% extends "template.njk" %}
```

Comment:
why is one file named subscription-case-search and other is flipped to subscription-search-case-results?
You are searching for a case reference number

Reply:
changed.